js/about/extensions.js

```diff
@@ -9,13 +9,51 @@ const ImmutableComponent = require('../components/immutableComponent')
 const messages = require('../constants/messages')
 const aboutActions = require('./aboutActions')
 
+const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('../../app/renderer/components/styles/global')
+const commonStyles = require('../../app/renderer/components/styles/commonStyles')
+
 const ipc = window.chrome.ipcRenderer
 
 // Stylesheets
-require('../../less/about/itemList.less')
-require('../../less/about/extensions.less')
+require('../../less/about/common.less')
 require('../../node_modules/font-awesome/css/font-awesome.css')
 
+const styles = StyleSheet.create({
```

Comment:
[minor] I think it's more likely to edit logic code more than editing styles, and this way we have to scroll down all Aphrodite stuff to get there. Can we put styles at the end of the file?